Kitsune/ClubPenguin/World.php

```diff
@@ -172,9 +174,9 @@
 	use Handlers\Play\PlayerTransformation;
 	use Handlers\Play\Music;
 	
-	use Handlers\Game\General;
-	use Handlers\Game\Multiplayer;
-	use Handlers\Game\Waddle;	
+	use Handlers\Game\Games;
+	use Handlers\Game\Four;
```

Comment:
Should probably move all implementation related code to it's own folder/namespace (Game\FindFour\xxx).